core/src/test/scala/unit/kafka/utils/JsonTest.scala

```diff
@@ -46,21 +46,21 @@ class JsonTest {
 
   @Test
   def testJsonEncoding() {
```

Comment:
You should rename this to `testLegacyEncodeAsString` and add new tests for the new methods.

Reply:
Done, pushing the changes.